setup.cfg

```diff
@@ -1,3 +1,20 @@
 [flake8]
 exclude = .git,venv,env
 max-line-length = 79
+
+[semantic_release]
+; publish every time you push to 'dev'
+branch = dev
```

Comment:
Perhaps we may want to change this to `master`, since that is really our release branch. This allows us to hold back multiple API-breaking commits and release them together. Would be annoying to have a breaking change in every third commit or so.